flycheck.el

```diff
@@ -3617,6 +3617,19 @@ IDs, that include the name of the tool."
   (seq-do (lambda (err) (setf (flycheck-error-id err) nil)) errors)
   errors)
 
+(defun flycheck-fill-empty-line-numbers (errors)
+  "Set ERRORS without lines to line 0.
+
+Use as `:error-filter' for syntax checkers that output errors
+without line numbers.
+
+Return ERRORS."
+  (seq-do (lambda (err)
+            (unless (flycheck-error-line err)
+              (setf (flycheck-error-line err) 0)))
+          errors)
```

Comment:
All of the other filters use `seq-do`, so I am inclined to keep the code consistent.